src/app/shared/validationMessages.model.ts

```diff
@@ -67,25 +67,25 @@ export const exampleDescriptorPatterns = {
 };
 
 export const validationDescriptorPatterns = {
-  gitPath: '^([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)/([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)$',
-  smkPath: '^/([^/?:*|<>]++/)*(Snakefile|[^./?:*|<>]++.smk))$',
-  cwlPath: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(cwl|yaml|yml)',
-  wdlPath: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.wdl$',
-  nflPath: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(config)',
-  dockerfilePath: '^/([^/?:*|<>]+/)*(([a-zA-Z]+[.])?Dockerfile|Dockerfile([.][a-zA-Z]+)?)$',
-  testFilePath: '^/([^/?:*|<>]+/)*[^/?:*|<>]+.(json|yml|yaml)$',
-  imagePath: '^(([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)|_)/([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)$',
-  repoNameWithSlashesImagePath: '^(([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)|_)/([a-zA-Z0-9]+([-_./][a-zA-Z0-9]+)*)$',
-  privateAmazonImagePath: '^_/([a-zA-Z0-9]+([-_./][a-zA-Z0-9]+)*)$', // Has an empty namespace. Allows for slashes in repo name
+  gitPath: '^([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)\\/([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)$',
+  smkPath: '^\\/([^\\/?:*\\|<>]++\\/)*(Snakefile|[^.\\/?:*\\|<>]++.smk))$',
+  cwlPath: '^\\/([^\\/?:*\\|<>]+\\/)*[^\\/?:*\\|<>]+.(cwl|yaml|yml)',
+  wdlPath: '^\\/([^\\/?:*\\|<>]+\\/)*[^\\/?:*\\|<>]+.wdl$',
+  nflPath: '^\\/([^\\/?:*\\|<>]+\\/)*[^\\/?:*\\|<>]+.(config)',
+  dockerfilePath: '^\\/([^\\/?:*\\|<>]+\\/)*(([a-zA-Z]+[.])?Dockerfile|Dockerfile([.][a-zA-Z]+)?)$',
+  testFilePath: '^\\/([^\\/?:*\\|<>]+\\/)*[^\\/?:*\\|<>]+.(json|yml|yaml)$',
+  imagePath: '^(([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)|_)\\/([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)$',
+  repoNameWithSlashesImagePath: '^(([a-zA-Z0-9]+([-_.][a-zA-Z0-9]+)*)|_)\\/([a-zA-Z0-9]+([-_./][a-zA-Z0-9]+)*)$',
+  privateAmazonImagePath: '^_\\/([a-zA-Z0-9]+([-_./][a-zA-Z0-9]+)*)$', // Has an empty namespace. Allows for slashes in repo name
```

Comment:
Needs an escape on the slash in the character class (inside the square brackets)?